src/lib/userStorage/FeedStorage.js

```diff
@@ -58,7 +59,7 @@ export const TxTypeToEventType = {
   TX_REWARD: FeedItemType.EVENT_TYPE_BONUS,
   TX_MINT: FeedItemType.EVENT_TYPE_RECEIVE,
   TX_BRIDGE_OUT: FeedItemType.EVENT_TYPE_SENDBRIDGE,
-  TX_BRIDGE_IN: FeedItemType.EVENT_TYPE_RECEIVE,
+  TX_BRIDGE_IN: FeedItemType.EVENT_TYPE_RECEIVEBRIDGE,
```

Comment:
1. because we use a pretty simple regex to determine send/receive transactions. sendBridge would not match
https://github.com/GoodDollar/GoodDAPP/blob/577847a9534ed79467bdea12c30995de797696a3/src/lib/wallet/utils.js#L279

Reply:
1. I ment if we have a specific event to sendbridge and I see we do
2. yes it is simpler because changing the event type can cause unforseen bugs. How is the check reason==='bridge' is different than type==='recievebridge' checking the addresses should be the second check and is only slightly more expensive

Reply:
yes, and I also realized it's simpler just because it does not require updating old feed-items

Reply:
>  believe the simplest way is to use the "bridgeAddresses" 

exactly. we don't have event type for claim, it's "received" but from GD UBI contract address